### PR TITLE
Update IMS handling table to include INM

### PR DESCRIPTION
**Pull request by mlibbey**

### CODE REVIEW

**Comment by shukitchan** [comment]
Ha. That's useful. I don't know we have such table. Should we put this in a doc as well?

**Comment by zwoop** [comment]
Hmm, this clearly shouldn't fail ... you might have to rebase with current master, but lets try another build [approve ci].

**Comment by mlibbey** [comment]
> Ha. That's useful. I don't know we have such table. Should we put this in a doc as well?

Sure -- want to suggest a location for it?

**Comment by shukitchan** [comment]
perhaps a section under here ? https://docs.trafficserver.apache.org/en/latest/admin-guide/storage/index.en.html

File: doc/admin-guide/storage/index.en.rst [hunk 1]
```diff
@@ -349,9 +349,38 @@ The next time Traffic Server receives a request for the removed object,
...
+``400 Malformed Pushed Response Header`` message is returned.
+
+To push an object, first save the object headers and body into a file. For instance: ::
+
+      $ curl -s -i -o /path/to/file "http://example.com/push_me.html"
+      $ cat /path/to/file
+      HTTP/1.1 200 OK
+      Date: Wed, 31 May 2017 16:01:59 GMT
+      Access-Control-Allow-Origin: *
+      Cache-Control: max-age: 10800
```
Comment (shukitchan) [inline doc/admin-guide/storage/index.en.rst]: should this be max-age=10800 ?

Comment (shukitchan) [inline doc/admin-guide/storage/index.en.rst]: should this be max-age=10800 ?

**Comment by mlibbey** [comment]
@shukitchan updated.

**Comment by bryancall** [comment]
[approve ci docs]

**Comment by mlibbey** [comment]
cherry-picked to 7.1.x

**Pull request merged**